lib/bsdlib/bsd_os.c

```diff
@@ -51,7 +51,9 @@ void IPC_IRQHandler(void);
 
 int32_t bsd_os_timedwait(uint32_t context, uint32_t timeout)
 {
-	/* TODO: to be implemented */
+
+	k_sleep(K_MSEC(10));
```

Comment:
This should not be part of this PR. Or create a new commit at least.

Reply:
I could splilt out the example in a separate PR.